### ISSUE TITLE
[a11y]: aria-label attributes

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/page-constructor#482]
There are some elements that should have `aria-label` attribute for better understanding the wbsite content with screenreaders. At least:

1. The cross icon, appearing after some value entered in the search field, should be labelled with `aria-label`

1. Add `aria-label="Close"` for close button in FullscrrenImage

1. Need to set aria-label for Play, Pause and Mute buttons for [Media](https://preview.yandexcloud.dev/page-constructor/?path=/story/components-pics-video-datalens-media--youtube) component when `controls=”custom”`. Not forget that aria-label is different for different languages.

**Comment by Kyzyl-ool** [comment]
Moved tasks from this issue here: https://github.com/gravity-ui/page-constructor/issues/486
